sound/soc/sof/sof-priv.h

```diff
@@ -307,6 +307,7 @@ struct snd_sof_pcm_stream {
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
 	bool d0i3_compatible; /* DSP can be in D0I3 when this pcm is opened */
+	bool d0i3_active; /* This stream is active at D0I3 */
```

Comment:
you are not setting this flag in this PR, so it's hard to figure out what it really represents.

You are really confusing concepts and terminology

a device is active/suspended
a stream is opened/prepared/triggered.

an active stream is ambiguous.

Reply:
I think in ALSA concept, we also have 'active' concept for a PCM stream, where when the stream is in opened or prepared or running, haven't we? I didn't use 'd0i3_opened' as it sounds confusing to me, or what wording do you suggest here?